Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4371,7 +4371,11 @@ def to_context(self):
                 outputs_key = '{}'.format(self.outputs_prefix)
                 outputs[outputs_key] = self.outputs
             else:
```

Comment:
This code will also handle an edge case if outputs provided as non dict type (string, int, lists), and without outputs_prefix - because EntryContext must be dict
```suggestion
            elif not isinstance(self.outputs, dict) and not self.outputs_prefix:
                raise ValueError('outputs_prefix is missing')
            else:
```

Reply:
moved it to the `__init__` as it's a prerequisite. We might consider making a function that asserts all those changes.